baselines/diabetic_retinopathy_detection/deterministic.py

```diff
@@ -0,0 +1,272 @@
+# coding=utf-8
+# Copyright 2020 The Uncertainty Baselines Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ResNet-50 on Kaggle's Diabetic Retinopathy Detection trained with
+maximum likelihood and gradient descent."""
+
+import os
+import time
+
+from absl import app
+from absl import flags
+from absl import logging
+
+import tensorflow as tf
+import uncertainty_baselines as ub
+import utils  # local file import
```

Comment:
is it possible to change this to `from uncertainty_baselines.baselines.diabetic_retinopathy_detection import utils`?

Reply:
Is uncertainty_baselines.baselines supposed to be available as a module? Other imports in the repo seem to only access the uncertainty_baselines subdirectory, and the ['datasets', 'models', 'optimizers', 'strategy_utils', 'utils'] modules